Loop/Public/FeedbackLoop.swift

```diff
@@ -81,7 +81,7 @@ extension Loop {
         ///             and having them consumed by `output` using the `SignalProducer.enqueue(to:)` operator.
         public init(
             events: @escaping (
-                _ state: SignalProducer<State, Never>,
+                _ state: SignalProducer<(State, Event?), Never>,
```

Comment:
```suggestion
                _ statesAndEvents: SignalProducer<(updated: State, by: Event?), Never>,
```

inspired by https://developer.apple.com/documentation/swift/dictionary/3127161-init